myst_nbs/case_studies/spline.myst.md

```diff
@@ -11,12 +11,13 @@ kernelspec:
   name: python3
 ---
 
-# Splines in PyMC3
+(spline)=
+# Splines
 
-:::{post} May 6, 2022 
-:tags: patsy, pymc.Deterministic, pymc.Exponential, pymc.Model, pymc.Normal, regression, spline
+:::{post} June 4, 2022 
+:tags: patsy, regression, spline
 :category: beginner
-:author: Joshua Cook updated by Tyler James Burch, Chris Fonnesbeck
+:author: Joshua Cook, Tyler James Burch, Chris Fonnesbeck, Benjamin Vincent
```

Comment:
I am completely unaware of how/if we are enforcing this, especially during this last sprint week. I generally try to comment on that to make sure people who should but haven't yet add themselves and viceversa. According to what we decided and wrote in: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#authorship-and-attribution and last sentence in https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell. You should not add yourself to the metadata, we don't have a word defined for fixing typos and such, we could add one and add yourself to the authorship section

Reply:
Want quite sure about the meta data versus the authorship at the end.  Totally happy to remove myself.